application/src/main/java/org/thingsboard/server/service/cloud/CloudManagerService.java

```diff
@@ -400,6 +414,10 @@ private void updateConnectivityStatus(boolean activityState) {
     private void scheduleReconnect(Exception e) {
         initialized = false;
 
+        if (queueType.equals("kafka")) {
+            cloudEventService.unsubscribeConsumers();
```

Comment:
Maybe let's consider changing this name to clean up (another way), so in PostgresCloudEventService we will leave it empty and won't face this check.. Also it's unclear why unsubscribeConsumers exists in PostgresCloudEventService - that's my point